vega_sim/api/data.py

```diff
@@ -827,3 +827,24 @@ def get_trades(
             )
         )
     return res_trades
+
+
+def get_network_parameter(
+    data_client: vac.VegaTradingDataClientV2,
+    key: str,
+) -> Any:
```

Comment:
Does this not always return a string? (Or at worst a string/float/int?)

Reply:
I'm also on the fence on whether this should be in `data_raw` (because we aren't processing the output at all) or `data` (because we're unlikely to want to)

Reply:
It also might be cool to have a `to_type` (or similar name) arg which takes a function and automatically converts the result to that (so, you don't have to do it externally) e.g. `foo = get_network_parameter(key='bar', to_type=int)`. It'll make the type signature marginally funkier but that's a challenge to the reader

Reply:
> Does this not always return a string? (Or at worst a string/float/int?)

Not sure which, but one of those is definitely right! I'll have a look and then change the typing 👍 

> It also might be cool to have a to_type (or similar name) arg which takes a function and automatically converts the result to that (so, you don't have to do it externally) e.g. foo = get_network_parameter(key='bar', to_type=int). It'll make the type signature marginally funkier but that's a challenge to the reader


I like that `to_type` idea as well, I'll add that in. When I do I guess we can treat that as processing the output and move it into `data_raw`?